sources/Renderer/DXCommon/DXCore.cpp

```diff
@@ -15,6 +15,7 @@
 #include <stdexcept>
 #include <algorithm>
 #include <d3dcompiler.h>
+#include <dxcapi.h>
```

Comment:
I presume this header is included in the Windows SDK, so let me add a new CMake option to exclude it from builds that don't have that SDK installed. Same goes for the code that uses it. Eventually, I will probably move it out into a separate file to have build specific code nicely separated instead of having several `#ifdef` preprocessor blocks.

Reply:
It is included with the Windows Kits, like `d3dcompiler.h`, and it should exist alongside any copy of the d3d12 headers.

Reply:
I believe `d3dcompiler.h` and `d3d12.h` are included with Visual Studio, but maybe that changed with newer Visual Studio editions; I am still working with VS 2015.

Reply:
Given the relative newness of SM6 and the DXC, it sounds like you're right - it's changed as of a recent version of the Windows SDK. Are you unable to build, or is the dependency missing?

As a note, the header for the DXC is available at the repository for it ([here](https://github.com/microsoft/DirectXShaderCompiler)) and should work unconditional of SDK version or compiler, if included as a submodule.

Reply:
I see the problem now, my CMake version does not select the latest Windows SDK I have installed on my machine. I can work that around by manually selecting it in VS for now.